components/AddForm/AddForm.spec.tsx

```diff
@@ -39,7 +39,7 @@ const flexibleForms: [string, boolean, boolean][] = [
   ['FACE overview assessment', false, false],
   ['Safeguarding Adult Concern Form', false, false],
   ['Safeguarding adult manager decision on concern', false, false],
-  ['Child Case Note', false, false],
+  ['Case note', false, false],
```

Comment:
we're replacing this component with `<AddFormDialog/>` shortly. should adjust this if that gets merged first